SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -29,7 +29,17 @@ namespace SpatialGDKServicesConstants
 	const FString SpotExe = CreateExePath(GDKProgramPath, TEXT("spot"));
 	const FString SchemaCompilerExe = CreateExePath(GDKProgramPath, TEXT("schema_compiler"));
 	const FString SpatialOSDirectory = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial/")));
-	const FString SpatialOSRuntimePinnedVersion("14.5.1");
 	const FString SpatialOSConfigFileName = TEXT("spatialos.json");
 	const FString ChinaEnvironmentArgument = TEXT(" --environment=cn-production");
+
+	const FString SpatialOSRuntimePinnedStandardVersion         = TEXT("0.4.0-preview-5");
+	const FString SpatialOSRuntimePinnedCompatbilityModeVersion = TEXT("14.5.1");
+
+	const FString InspectorURL   = TEXT("http://localhost:31000/inspector");
+	const FString InspectorV2URL = TEXT("http://localhost:31000/inspector-v2");
+
+	const FString PinnedStandardRuntimeTemplate               = TEXT("n1standard4_std40_action1g1");
```

Comment:
This doesn't look like a squid template - have they changed their naming standards?

Reply:
Yup - new template formats.